azurecontainerapps/src/TelemetryHelper.ts

```diff
@@ -0,0 +1,67 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { Utility } from './Utility';
+
+const ORYX_CLI_IMAGE: string = "mcr.microsoft.com/oryx/cli:debian-buster-20230207.2";
+
+export class TelemetryHelper {
+    readonly disableTelemetry: boolean = false;

```

Comment:
Nit: please consider not assigning the value: it should always be overwritten in any case, and gives us the security that if the code changes this value won't be used by mistake.

Reply:
Good call -- changing this.

Reply:
Thanks!